src/nodes/chunk_append/exec.c

```diff
@@ -786,6 +831,8 @@ constify_param_mutator(Node *node, void *context)
 			{
 				ExprContext *econtext = GetPerTupleExprContext(estate);
 				ExecSetParamPlan(prm.execPlan, econtext);
+				// reload prm as it may have changed
```

Comment:
When does it change? That would be more than one loops for chunk append plan, so I guess when it's on the inner side of a nested loop join?

Reply:
Its changed by the ExecSetParamPlan call above. I updated the column to be more clear.